fuzz/fuzz_targets/blake3.rs

```diff
@@ -1,28 +1,75 @@
 #![no_main]
 
+use std::iter::IntoIterator;
 use std::sync::LazyLock;
 
+use arbitrary::Arbitrary;
 use bitcoin::ScriptBuf;
 use bitcoin_script_stack::optimizer;
 use libfuzzer_sys::fuzz_target;
-
 use bitvm::execute_script_buf;
 use bitvm::hash::blake3::{
     blake3_compute_script, blake3_push_message_script, blake3_verify_output_script,
 };
 
-static BLAKE3_COMPUTE_SCRIPT: LazyLock<ScriptBuf> =
-    LazyLock::new(|| optimizer::optimize(blake3_compute_script(32).compile()));
+static BLAKE3_COMPUTE_SCRIPTS: LazyLock<[ScriptBuf; 4]> =
+    LazyLock::new(|| [64, 128, 128, 448]
```

Comment:
Third element of this array is wrong.

Reply:
Thanks for pointing this out. This typo is so easy to miss. I pushed a fix.

Reply:
The danger of magic numbers!